metrics/cli/docs/features/doctor_output_improvements/06_doctor_output_improvements.md

```diff
@@ -0,0 +1,118 @@
+
+To admit that this is possible to implement the feature, let's take a look at the existing examples:
+
+- Flutter provides a similar `doctor` command that validates the environment to be ready for development using Flutter SDK. The command output is clear and useful and provides the conclusion of the command run summarizing results.
+     
+     ![Flutter Doctor](images/flutter_doctor_example.png)
+
+- The CI Integrations `validate` command consumes the configuration file and validates its fields resulting in a clear output with conclusions for each field.
+
+     ![CI Integrations Validate](images/ci_integrations_validate_example.png)
+
+Therefore, the feature implementation is possible since the real-case examples exist.
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+The `doctor` command performs the set of checks that validates whether the machine has the required third-party tools installed. Each check stands for a single validation item for the environment the Metrics CLI runs in. The `doctor` command output consists of a set of results for each validation item performed during the command run. Therefore, we can state that the command itself is a validation for the environment. This makes the `doctor` command similar to the CI Integrations `validate` command for [Config Validator](https://github.com/platform-platform/monorepo/blob/master/metrics/ci_integrations/docs/04_ci_integrations_config_validator.md). Let's define the requirements for the output improvements feature:
+
+- A single check shouldn't output log messages to the user's console to avoid confusion.
+- A single check shouldn't output errors to the user's console to avoid confusion.
+- Single check output should start with a leading box with an indicator that clearly defines the result of the validation.
+     - The successful validation should start with the mark sign in brackets: `[✓]`.
+     - The failed validation should start with the cross sign in brackets: `[✗]`.
+     - The validation that finishes successfully but has warnings should start with the exclamation sign in brackets: `[!]`.
+     - The validation with an unknown result should start with the question mark in brackets: `[?]`.
+- Single check output should contain the validation item name or its description to define the target of validation.
+- Single check output should contain the validation result description (could be empty).
+- A single check should contain the additional output of the appropriate process. This output is optional by default for the successful checks and is required for fail, warning, and unknown results.
+- A single check additional output should be human-readable and highlighted from the main result according to the following rules:
```

Comment:
🏷️
```suggestion
- A single check additional output should be human-readable and decoupled from the main result according to the following rules:
```
It seems like the output is decoupled from the check result since it has more indent. What do you think?